test/bench/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ target_sources(
     bench.cpp
     helpers.hpp
     synthetic_benchmarks.cpp synthetic_benchmarks.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/../statetest/statetest_loader.cpp
```

Comment:
The statetest loader should be a separate support library. We should make this change before this PR. Let me know if you want to work on this. If not I can make the change quickly.

Reply:
I'll propose a new pr.